src/tests/banners/SubscriptionsBannerTest.ts

```diff
@@ -0,0 +1,17 @@
+import { BannerPageTracking, BannerTargeting, BannerTest } from '../../components/BannerTypes';
+import { readerRevenueRegionFromCountryCode } from './bannerSelection';
+
+export const SubscriptionsBannerPath = 'subscriptions-banner.js';
+
+export const SubscriptionsBanner: BannerTest = {
```

Comment:
I feel like if we're going to have a `WeeklyBanner` then this should be `DigitalSubscriptionBanner` as that is the product name

Reply:
Yes, definitely, thanks for pointing this out.